edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html

```diff
@@ -0,0 +1,148 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions message-top-bar" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
```

Comment:
@chrisndodge Per discussion yesterday with @griffresch and @marcotuts we are suggesting changes to this template so that it has more of a status and next steps flow, like seq_proctored_exam_ready_to_start.html.